src/dom/Draggable.js

```diff
@@ -99,13 +99,16 @@ L.Draggable = L.Evented.extend({
 		this._moving = true;
 
 		L.Util.cancelAnimFrame(this._animRequest);
-		this._animRequest = L.Util.requestAnimFrame(this._updatePosition, this, true, this._dragStartTarget);
+		var animationCallback = function () {
+			this._updatePosition(e);
+		};
+		this._animRequest = L.Util.requestAnimFrame(animationCallback, this, true, this._dragStartTarget);

```

Comment:
You can use `L.bind(animationCallback, null, e)` instead. Or `this._lastEvent` to avoid closure entirely.